detection-rules/attachment_microsoft_image_lure_qr_code.yml

```diff
@@ -19,12 +19,35 @@ source: |
                 regex.icontains(.scan.ocr.raw, 'scan|camera')
                 and regex.icontains(.scan.ocr.raw, '\bQR\b|Q\.R\.|barcode')
             )
-          )
-          or (
-            any(file.explode(.),
-                .scan.qr.type == "url"
-                // recipient email address is present in the URL, a common tactic used in credential phishing attacks 
-                and any(recipients.to, strings.icontains(..scan.qr.data, .email.email))
+            or (
+              any(file.explode(.),
+                  .scan.qr.type == "url"
+                  // recipient email address is present in the URL, a common tactic used in credential phishing attacks 
+                  and any(recipients.to,
+                          strings.icontains(..scan.qr.data, .email.email)
+                          or (
```

Comment:
don't need the parentheses here `x or (y or z)` can be flattened to `x or y or z`

Reply:
Should be fixed!